metadata.yaml

```diff
@@ -0,0 +1,8 @@
+service:
+  full_name: "Amazon API Gateway"
+  short_name: "APIGWv2"
+  link: "https://aws.amazon.com/api-gateway/"
+  documentation: "https://docs.aws.amazon.com/apigateway/latest/developerguide/welcome.html"
```

Comment:
shouldn't this be pointing to api gateway "v2" instead? /cc @vijtrip2

Reply:
no the service and docs is still "apigateway". apigatewayv2 is just subservice used for a certain product of apigateway.

Reply:
Ohhh gotcha! thanks!